notebooks/CalculateSharpness.ipynb

```diff
@@ -830,7 +830,7 @@
     "from getpass import getpass\n",
     "readAnnotateConn = BlitzGateway(\n",
     "    raw_input(\"Username: \"), getpass(\"OMERO Password: \"),\n",
-    "    host=\"tadpole.openmicroscopy.org\")\n",
+    "    host=raw_input(\"Hostname: \"))\n",
```

Comment:
Shouldn't this use the IDR_ENV variables?

Reply:
This section is about writing back to a server so I do not think we want it to point to the production IDR server. Do you have other suggestions for the name?

Reply:
My understanding is that the optional writing section was  introduced on purpose in the initial notebook  - see https://github.com/IDR/idr-notebooks/pull/18. Let us discuss with @bramalingam on return on the best way to handle any confusion.